_components/banner/guidance/variants/default/when-to-consider-something-else.md

```diff
@@ -0,0 +1,2 @@
+- **If you don't use a .gov/.mil domain and HTTPS.** The Design System's banner text identifies `.gov` and `.mil` domains and HTTPS as indicators that a website is an official government website. Use this banner only if your site uses both the proper top-level domain (TLD) _and_ HTTPS.
```

Comment:
Not sure if this matters, but you might prefer the apostrophe character `’` rather than the single quote `'`. If so, there are a few other places in this PR that use the single quote character in a contraction.

```suggestion
- **If you don't use a .gov/.mil domain and HTTPS.** The Design System's banner text identifies `.gov` and `.mil` domains and HTTPS as indicators that a website is an official government website. Use this banner only if your site uses both the proper top-level domain (TLD) _and_ HTTPS.
```

Reply:
I might and indeed do prefer a real apostrophe, but I'd bet this isn't the only page where that happens so I think let's leave it for now and we'll do a bigger find-replace to fix it everywhere.